server/src/test/java/org/candlepin/controller/PoolManagerTest.java

```diff
@@ -1703,117 +1703,6 @@ public void createPoolsForPoolBonusExist() {
         assertThrows(IllegalStateException.class, () -> pRules.createAndEnrichPools(p, existingPools));
     }
 
-    @Test
-    public void testFabricateSubscriptionFromPool() {
-        Product product = TestUtil.createProduct("product", "Product");
-        Product provided1 = TestUtil.createProduct("provided-1", "Provided 1");
-        Product provided2 = TestUtil.createProduct("provided-2", "Provided 2");
-        product.setLocked(true);
-        provided1.setLocked(true);
-        provided2.setLocked(true);
-
-        ProductData productDTO = product.toDTO();
-        ProductData provided1DTO = provided1.toDTO();
-        ProductData provided2DTO = provided2.toDTO();
-
-        Pool pool = mock(Pool.class);
-
-        HashSet<Product> provided = new HashSet<>();
-        HashSet<ProductData> providedDTOs = new HashSet<>();
-        provided.add(provided1);
-        provided.add(provided2);
-        providedDTOs.add(provided1DTO);
-        providedDTOs.add(provided2DTO);
-
-        Long quantity = new Long(42);
-
-        Date startDate = new Date(System.currentTimeMillis() - 86400000);
-        Date endDate = new Date(System.currentTimeMillis() + 86400000);
-        Date updated = new Date();
-
-        String subscriptionId = "test-subscription-1";
-
-        when(pool.getOwner()).thenReturn(owner);
-        when(pool.getProduct()).thenReturn(product);
-        when(mockProductCurator.getPoolProvidedProductsCached(any(Pool.class))).thenReturn(provided);
-        when(pool.getQuantity()).thenReturn(quantity);
-        when(pool.getStartDate()).thenReturn(startDate);
-        when(pool.getEndDate()).thenReturn(endDate);
-        when(pool.getUpdated()).thenReturn(updated);
-        when(pool.getSubscriptionId()).thenReturn(subscriptionId);
-        // TODO: Add other attributes to check here.
-
-        Subscription fabricated = manager.fabricateSubscriptionFromPool(pool);
-        pool.populateAllTransientProvidedProducts(mockProductCurator);
-        assertEquals(owner, fabricated.getOwner());
-        assertEquals(productDTO, fabricated.getProduct());
-        assertEquals(providedDTOs, fabricated.getProvidedProducts());
-        assertEquals(quantity, fabricated.getQuantity());
-        assertEquals(startDate, fabricated.getStartDate());
-        assertEquals(endDate, fabricated.getEndDate());
-        assertEquals(updated, fabricated.getModified());
-        assertEquals(subscriptionId, fabricated.getId());
-    }
-
-
-    /**
-     * See BZ1292283
-     */
-    @Test
-    public void testFabricateSubWithMultiplier() {
```

Comment:
One last thing to mention:

I think we should preserve these 3 unit tests (this one, along with `testFabricateSubWithZeroInstanceMultiplier` and `testFabricateSubWithMultiplierAndInstanceMultiplier`) because they test the quantity translation logic. We can port them directly into `PoolToSubscriptionTranslatorTest.java` and change the fabricateSubscriptionFromPool call to `translator.translate(pool, SubscriptionDTO)`

Reply:
Added these tests under `PoolToSubscriptionTranslatorTest.java`